src/java.base/share/classes/sun/security/tools/keytool/resources/keytool_de.properties

```diff
@@ -27,7 +27,7 @@ NEWLINE=\n
 STAR=*******************************************
 STARNN=*******************************************\n\n
 # keytool: Help part
-.OPTION.=\u0020[OPTION]...
+.OPTION.=\ [OPTION]...
```

Comment:
We spoke about this offline. This is fine, property file values need to either start with a `\ `or `\u0020` to denote intentional leading white space. In this case, since we convert to UTF-8 native characters from escape sequences this gets included as well. Functionally, they are equivalent and the new form works well with the translation process.